### ISSUE TITLE
make me an application

### ISSUE THREAD

**Issue by thigg** [sailfishos-chum/kasts#1]
Kasts is not visible in chum when not selecting show all

**Comment by thigg** [comment]
Should the type be desktop-application like in angelfish?

**Comment by piggz** [comment]
Yes it should.  Also, i think some deps are missing from the .spec
